src/tribler-core/tribler_core/requirements.txt

```diff
@@ -0,0 +1,23 @@
+psutil
+configobj
+PyOpenSSL
+chardet
+libnacl
+pyasn1
+lz4
+pony
+service-identity
+decorator
+networkx
+cryptography
+cryptography
```

Comment:
```suggestion
cryptography
```

**suggestion/nit:** You could order all `requirements.txt` files alphabetically to make them easier to parse by humans.

Reply:
Thank you 👍  Implemented.